Solutions/Claroty/Hunting Queries/ClarotyUnapprovedAccess.yaml

```diff
@@ -0,0 +1,24 @@
+id: de0fca32-85f3-45df-872e-41e980e5d8d3
+name: Claroty - Unapproved access
+description: |
+  'Query searches for unaprooved access events.'

```

Comment:
Please correct the typo in the spelling of unapproved.

Reply:
fixed